modules/context/context.go

```diff
@@ -96,505 +79,6 @@ func (ctx *Context) TrHTMLEscapeArgs(msg string, args ...string) string {
 	return ctx.Tr(msg, trArgs...)
```

Comment:
call `Locale.Tr` explicitly

```suggestion
	return ctx.Locale.Tr(msg, trArgs...)
```

The "context" is too complex, IMO calling it explicitly seems better.